third_party/2and3/pytz/__init__.pyi

```diff
@@ -11,7 +11,7 @@ country_timezones = ...  # type: Dict
 country_names = ...  # type: Dict
 
 
-class UTC(dt.tzinfo):
+class __UTC(dt.tzinfo):
```

Comment:
I'm not super keen on the `__` prefix, given its conventional meaning. Maybe just name the class something longer with a single `_`, e.g. `_UTCclass`?